scripts/telegram_ros_bridge

```diff
@@ -141,6 +149,46 @@ class TelegramROSBridge(object):
             self._telegram_updater.bot.send_photo(self._telegram_chat_id,
                                                   photo=StringIO(cv2.imencode('.jpg', cv2_img)[1].tostring()))
 
+    def _telegram_location_callback(self, _, update):
+        """
+        Called when a new telegram Location is received. The method will verify whether the incoming Location is
+        from the bridges telegram conversation by comparing the chat_id.
+        :param update: Received update that holds the chat_id and message data
+        """
+        location = update.message.location
+        if isinstance(location, Location):
+            rospy.loginfo("Incoming location from telegram: {}".format(location))
+            if self._telegram_chat_id is None:
+                rospy.logwarn("Discarding message. No active chat_id.")
+                update.message.reply_text("ROS Bridge not initialized. Type /start to set-up ROS bridge")
+            elif self._telegram_chat_id != update.message.chat_id:
+                rospy.logwarn("Discarding message. Invalid chat_id")
+                update.message.reply_text(
+                    "ROS Bridge initialized to another chat_id. Type /start to connect to this chat_id")
+            else:
+                navsatfix = NavSatFix()
+                navsatfix.header.stamp = update.message.date # TODO convert tot ROS Time
```

Comment:
Can be removed I guess

Reply:
Hey, I thought I fixed this. I do want to include the timestamp the user sent the location

Reply:
Ah, I see now, done correctly in the lines below. 

I made all this on the train with laptop battery dying so i had to be quick :-)